src/logit.c

```diff
@@ -166,6 +167,27 @@ void logit(void) {
 		    callreturn = 0;
 		} else if (defer_store > 1) {
 		    if ((cqmode == CQ) && iscontest) {
+			if (cqmode == CQ && resend_call != RESEND_NOT_SET) {
```

Comment:
Let's move this block to a separate function in order to keep code readable.

Reply:
Commit 44e73c7 fixes. Please resolve this conversation if you agree the modification